flip.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @module 101/flip
+ */
+
+/**
+ * Returns a function with flipped arguments
+ * @function module:101/flip
+ * @param {function} f - function to be flipped
+ * @return {function}
+ */
+module.exports = flip;
+
+function flip(f) {
+  return function() {
+    var args = [].slice.call(arguments);
```

Comment:
very much a nit, but to be consistent with the rest of the code base, could you swap out `[].slice` for `Array.prototype.slice`?